docs/user-guide/ze-usage.md

```diff
@@ -75,8 +75,43 @@ If Zowe Explorer is not installed, you can install the current release of the ex
 2. In the **Side Bar**, search for `Zowe Explorer`. Click the **Install** button on the Zowe Explorer search result item. This opens a Zowe Explorer tab in the **Editor** area.
 
    ![Zowe Explorer search result item](../images/ze/ZE-zowe-explorer-result-item.jpg "Zowe Explorer search result item")
-<br /><br />
-
+<br/><br/>
 
 3. Click the **Down** arrow next to the **Uninstall** button. Select **Install Another Version…** to open a dropdown menu that lists previous versions of Zowe Explorer.
 4. Search for and click the version of Zowe Explorer you want to install.
+
+## Credentials in Zowe Explorer
+
+When working in remote or virtualized environments &mdash; such as Eclipse Che, GitHub Codespaces, CodeReady Workspaces &mdash; administrators may find the configuration process for storing credentials securely too cumbersome. Instead, they may prefer to rely on the security tools integrated with these environments, such as file access permissions. To do so, administrators need to disable Zowe Explorer's credential management functionality.
+
+### Preventing Zowe Explorer from storing credentials
+
+1. Open the `zowe.config.json` file in Visual Studio Code.
+
+2. Find the `autoStore` property.
+3. Set the `autoStore` property to `false`.
+
+   Credentials will be stored on a per profile/per panel basis until one of the following takes place:
+
+   - Data Sets/USS/Jobs panel refresh caused by an update to the `zowe.config.json` file
```

Comment:
I know we have gone back and forth on this, but should we say panels, trees, sections, ...?
Panels may be confusing since this is what VSCode has defined as panels:
- ![image](https://github.com/zowe/docs-site/assets/37381190/063134b3-8576-4556-9d8c-0902678aa476)
 
How about `Side Bar Section`?

Reply:
Using "trees"